webapp/src/ts/app.component.ts

```diff
@@ -633,14 +633,38 @@ export class AppComponent implements OnInit {
       'boot_time:1:to_first_code_execution',
       window.startupTimes.firstCodeExecution - window.startupTimes.start
     );
+
+    if (window.startupTimes.replicationStarted && window.startupTimes.replicationEnded) {
```

Comment:
@dianabarsan I feel like incomplete if we don't log anything when it didn't replicate / purge.
Do you think it'd be helpful to log a "boot_time:0:no_replication" or "boot_time:2_1:to_replication"= false ?

Reply:
Replication should only happen once, after 1st login, so we can leave out "no-replication", because it just bloats. 
We should log if purging did or didn't happen.

Reply:
That works!

Reply:
Done